scripts/adjustEnvVars.sh

```diff
@@ -165,14 +168,9 @@ MAKE="$MAKE1 --jobs=$JOBS"
 
 KERNEL_NAME=$(uname -s | tr '[:upper:]' '[:lower:]')
 
-function rmStep(){
-  rm -rf "$@"
-  rmdir -p --ignore-fail-on-non-empty `dirname "$@"`
```

Comment:
`rmdir -p` remove recursively all the parent empty folders. This is needed in some of the builds since they are used as flagfiles too to prevent duplicated builds, and so they are using the `rmstep()` function.